.gitignore

```diff
@@ -16,6 +16,7 @@
 *.dylib
 examples/webapp/webapp
 examples/ginwebapp/ginwebapp
+examples/managementcli/managementcli
```

Comment:
@Bars92 do devs need to add this to go.work after this PR is merged? (I see I have the other examples  it in comment)